master/reference/public-cloud/aws.md

```diff
@@ -0,0 +1,115 @@
+
+To allow this BGP peering, ensure your security group settings for each
+calico host allows incoming tcp connections to port 179.
+
+You can check if your hosts have successfully established BGP sessions with
+one another using `calicoctl node status`.
+
+#### Allow Traffic Within a Single VPC Subnet
+
+By default, each EC2 instance performs source/destination checks on inbound
+and outbound traffic. This ensures that the instance is the source or
+destination of any traffic it sends or receives.
+
+Since Calico allocates a unique IP to each container using pools AWS is not aware
+of, container traffic will have an "unrecognized" source address and will be
+dropped as it passes through the ec2 hosts to the container.
+
+This reverse-path filtering is enabled by default, but can be disabled on each
+host, allowing calico traffic to pass through unaffected.
+**Ensure you follow
+[AWS documentation on disabling source/destination checks](http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_NAT_Instance.html#EIP_Disable_SrcDestCheck).**
+
+#### Allow Traffic Across Different Subnets
+
+Container traffic that traverses different subnets will past through
+a VPC routing table. If this VPC routing table does not know how to route
+Calico IPs, it will drop traffic. There are two ways two prevent this from
+happening:
+
+1. Masquerade Container Traffic
+
+   Container traffic can be masqueraded with its host IP so that the VPC
+   never sees the container IPs, allowing standard ec2 routing to take over.
+
+   Turn on traffic masquerading in pool settings by enabling:
+
+   - `ipip` for container-to-container traffic.
+   - `nat-outgoing` for container-to-ec2-instancce traffic.
+
+   See [pool configuration](todo) for information on how to enable this for pools.
+
+2. Configure the VPC Routing Table
+
+   Alternatively, the VPC routing table can be programmed with calico IP routes
+   so that it can route container traffic natively. This can be accomplished in two
+   ways.
+
+   1. Deploy [canal][canal] and select the [aws-vpc backend][aws-vpc-backend],
+      which interacts with AWS api's to automatically program the routing table.
+
+      >Note: Doing so will limit you to just 50 routes.
+
+   2. Manually program the routing table (not recommended).
+
+#### Allow Traffic Across Different VPCs
+
+AWS drops non-masqueraded container traffic when it passes through multiple VPC's.
+This behavior can not be turned off.
+
+To circumvent this issue, enable `ipip` and `nat-outgoing` on your pools.
+
+#### Allow Workload-to-WAN Traffic
+
+In a private cloud where operators maintain full control of the networking
+fabric, NAT'ing typically takes place at the edge router.
+
+AWS does not support NAT'ing outbound traffic from unknown sources (i.e. calico IPs).
+
+Fortunately, calico can perform NAT on outbound traffic on each compute node by
+simply enabling `nat-outgoing` on each pool.
+
+## Next Steps
```

Comment:
Not sure a "next steps" is appropriate for this doc - it's not really a step-by-step.